pkgs/development/ocaml-modules/ppx_tools/default.nix

```diff
@@ -5,6 +5,7 @@ let param =
     version = "6.4";
     sha256 = "15v7yfv6gyp8lzlgwi9garz10wpg34dk4072jdv19n6v20zfg7n1";
     useDune2 = true;
+    nativeBuildInputs = [cppo];
     buildInputs = [cppo];
```

Comment:
Is it a left over ?

Reply:
Can't remember but might be that there's a library component of `cppo` that was also needed?

Reply:
Ok I will do another round. I see other packages with the same issue.